index.html

```diff
@@ -13,6 +13,50 @@
   </head>
 
   <body>
-   
+    <div class="container">
+      <header>
+        <div class="header">
+          <div class="header-p">
+            <p class="name">Kevin Toro</p>
+          </div>
+          <div class="fa-bars-icon">
+            <i class="fa-solid fa-bars"></i>
+          </div>
+        </div>
+      </header>
+      <main>
+        <section class="section-1">
+          <div class="headline">
+            <div class="Primary-text">Hey There. I’m Toro</div>
+            <div class="Supporting-text">I’m a Software Developer</div>
+            <div class="Description">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don’t hestiate to contact me.
+            </div>
```

Comment:
This is a paragraph. So this should be encapsulated in a `p` tag, again for semantic purposes.